src/App.jsx

```diff
@@ -0,0 +1,60 @@
+import './assets/cssStyle/App.css'
+import './components/VeggyHeader/HeaderCart/CartList/CartList.css'
+import Header from "./components/VeggyHeader/VeggyHeader.jsx";
+import {useEffect, useState} from "react";
```

Comment:
відділяй пробілами. Завжди май таку звичку, якщо пишеш обьект в 1 рядок
```suggestion
import { useEffect, useState } from "react";
```